mozilla/blurts-server
# QA tool for custom data

# References:

Jira: MNTOR-3357, MNTOR-3424, MNTOR-3425
Figma:



# Description
Separated from mock endpoints, and added complete customization. Admin is able to turn on/off custom and real data, and is able to adjust each field of the custom breach and broker objects.

# Screenshot (if applicable)

Not applicable.

# How to test
- Perform db migrations
- Sign in with an admin account
- Enter the data
- Make sure necessary toggles are turned on

# Checklist (Definition of Done)
